pipenv/cli/command.py

```diff
@@ -744,9 +744,21 @@ def verify(state):
     default="",
     help="Only add requirement of the specified categories.",
 )
+@option(
+    "--root-only",
```

Comment:
Are we sold on the flag name `--root-only`  How about `--from-pipfile` ?

Reply:
@matteius No problem. I'll modify it.